src/deck-options.md

```diff
@@ -492,6 +510,9 @@ shows that repeating a card multiple times in a single day does not significantl
 contribute to long-term memory, so your time is
 better spent on other cards or a shorter study session.
 
+In the latest version of Anki you can let FSRS control short-term scheduling by leaving the (re)learning steps field empty.
```

Comment:
```suggestion
You can let FSRS control short-term scheduling by leaving the (re)learning steps field empty.
```

Reply:
This is an experimental feature, so I think we should either clearly mark it as such, or avoid mentioning it yet.

Reply:
I think we should mention it just to increase the number of people who use it, which could increase the rate at which bugs are discovered